src/main/java/com/iexec/sms/secret/applicationruntime/ApplicationRuntimeSecret.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms.secret.applicationruntime;
+
+import com.iexec.sms.secret.Secret;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@Entity
+public class ApplicationRuntimeSecret extends Secret {
+    @Id
+    @GeneratedValue(generator = "system-uuid")
+    @GenericGenerator(name = "system-uuid", strategy = "uuid")
+    private String id;
```

Comment:
You're right, we probably don't need it. There's the same issue on `Web3Secret` but I'm not sure we could remove it as it could break already uploaded secrets:
https://github.com/iExecBlockchainComputing/iexec-sms/blob/538def0fbf0436c6ed98e1bb6e1b7ae80235042a/src/main/java/com/iexec/sms/secret/web3/Web3Secret.java#L37